packages/post-rector/src/Rector/NameImportingPostRector.php

```diff
@@ -49,7 +49,8 @@ public function __construct(
 
     public function enterNode(Node $node): ?Node
     {
-        if (! $this->parameterProvider->provideParameter(Option::AUTO_IMPORT_NAMES)) {
+        $parameterProviderNames = $this->parameterProvider->provideParameter(Option::AUTO_IMPORT_NAMES);
```

Comment:
```suggestion
        $autoImportNames = $this->parameterProvider->provideParameter(Option::AUTO_IMPORT_NAMES);
```

Only the constant should be used probably. What do you think based on other values?

Reply:
I implemented to use constant value.